shared/stories/prop-providers.js

```diff
@@ -41,7 +41,7 @@ const WaitingButton = () => ({
 })
 
 const Avatar = (following: string[] = defaultFollowing, followers: string[] = defaultFollowers) => ({
-  Avatar: (ownProps: _Avatar.OwnProps) => _Avatar.mockOwnToViewProps(ownProps, following, followers, action),
+  Avatar: (ownProps: any) => _Avatar.mockOwnToViewProps(ownProps, following, followers, action),
```

Comment:
This should be `_Avatar.Props`, right?

Reply:
we shouldn't even use this mockOwnToView props thing, that should entirely be in here

Reply:
i switched that to _Avatar.Props and the return value of the mocking as any